src/main/java/org/cactoos/scalar/Or.java

```diff
@@ -34,6 +34,36 @@
 /**
  * Logical disjunction.
  *
+ * <p>This class can be effectively used to iterate through
+ * a collection, just like
+ * {@link java.util.stream.Stream#forEach(java.util.function.Consumer)}
+ * works:</p>
+ *
+ * {@code
+ * new Or(
+ *    new ProcOf<>(input -> System.out.printf("\'%s\' ", input) ),
+ *    new IterableOf<>("Mary", "John", "William", "Napkin")
+ * ).value(); // will print 'Mary' 'John' 'William' 'Napkin' to standard output
+ *            // the result of this operation is always false
+ * }
+ *
+ * <p>This class could be also used for matching multiple boolean
```

Comment:
@Vatavuk could you also please add a note somewhere that Or performs a short-circuit evaluation?